bitnami/logstash/README.md

```diff
@@ -144,6 +160,7 @@ The following tables lists the configurable parameters of the Logstash chart and
 | `podDisruptionBudget.minAvailable`         | Minimum number / percentage of pods that should remain scheduled                                                     | `1`                                                     |
 | `podDisruptionBudget.maxUnavailable`       | Maximum number / percentage of pods that may be made unavailable                                                     | `nil`                                                   |
 | `extraDeploy`                              | Array of extra objects to deploy with the release (evaluated as a template).                                         | `nil`                                                   |
+| `initContainers`                           | Add extra init containers                                                                                            | `[]`                                                    |
 
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
```

Comment:
Add a "Persistence" section as other charts do under "Configuration and installation details"